Formula/colima.rb

```diff
@@ -37,6 +37,16 @@ def install
     (fish_completion/"colima.fish").write Utils.safe_popen_read(bin/"colima", "completion", "fish")
   end
 
+  service do
+    environment_variables HOME: ENV["HOME"], PATH: std_service_path_env
+
+    run [opt_bin/"colima", "start"]
+    keep_alive true
```

Comment:
```suggestion
    run [opt_bin/"colima", "start"]
    keep_alive true
    environment_variables HOME: ENV["HOME"], PATH: std_service_path_env
```
Thanks; let's keep the `run` line at the top for consistency with other formulae. (No strong opinion on `keep_alive` second; it just seems to read nicer this way.)

Please squash this change into your existing commit, and use the commit message:
```
colima: add service block
```